powerline/segments/common/net.py

```diff
@@ -85,21 +85,16 @@ def internal_ip(pl, interface='auto', ipv=4):
 		'vboxnet':  -5,  # VirtualBox bridge interface       : vboxnet0
 	}
 
-	_interface_start_re = re.compile(r'^([a-z]+?)(\d|$)')
-
 	def _interface_key(interface):
-		match = _interface_start_re.match(interface)
-		if match:
-			try:
-				base = _interface_starts[match.group(1)] * 100
-			except KeyError:
-				base = 500
-			if match.group(2):
-				return base - int(match.group(2))
-			else:
-				return base
-		else:
-			return 0
```

Comment:
And need an update to tests (tests/test_python/test_segments.py, class TestNet) with new interface examples.

Reply:
(Specifically, the ones which make the current code fail; I do not think that your variant is the best one.)